settings.py

```diff
@@ -15,6 +15,8 @@
     }
 }
 
+AUTH_USER_MODEL = "db.User"
```

Comment:
The `AUTH_USER_MODEL` is correctly set to `db.User`, which aligns with the custom user model defined in your project.